diff --git a/dist/components/auth/auth-provider.d.ts b/dist/components/auth/auth-provider.d.ts
index 3f9116dc77b74f6e8a2c8a5491b2c7f69bc479ea..ae1978735fa68ad8d30759cea601e0844cb82d0e 100644
--- a/dist/components/auth/auth-provider.d.ts
+++ b/dist/components/auth/auth-provider.d.ts
@@ -7,8 +7,9 @@ declare module "@better-auth-ui/core" {
         /**
          * The auth client to use for the authentication context.
          * @remarks `AuthClient`
+         * overryde on client code
          */
-        authClient: AuthClient;
+        // authClient: AuthClient;
     }
     /** Widen `AdditionalField.label` to `ReactNode` in the React package. */
     interface AdditionalFieldRegister {
