diff --git a/dist/client/react/index.d.mts b/dist/client/react/index.d.mts
index 27b2673ac7235ee43b946c595019667edc4f6f05..9084a036f4d508c8c4b80ef4eaa0ca3fffb52d10 100644
--- a/dist/client/react/index.d.mts
+++ b/dist/client/react/index.d.mts
@@ -22,7 +22,7 @@ type ClientSession<Option extends BetterAuthClientOptions> = InferClientAPI<Opti
  * React client returned by `createAuthClient`.
  */
 type ReactAuthClient<Option extends BetterAuthClientOptions> = UnionToIntersection<InferResolvedHooks<Option>> & InferClientAPI<Option> & InferActions<Option> & {
-  hydrateSession: (session: NonNullable<ClientSession<Option>> | null) => void;
+  hydrateSession(session: NonNullable<ClientSession<Option>> | null): void;
   useSession: () => {
     data: ClientSession<Option>;
     isPending: boolean;
